CHANGELOG.next.toml

```
@@ -21,3 +21,9 @@ message = "Add a trailing slash to the URI `/latest/meta-data/iam/security-crede
 references = ["aws-sdk-rust#560", "smithy-rs#1487"]
 meta = { "breaking" = false, "tada" = false, "bug" = true }
 author = "rcoh"
+
+[[smithy-rs]]
```

This is a SDK feature rather than a Smithy feature.
```suggestion
[[aws-sdk-rust]]
```